doc/reference/tooling/tt_cli/play.rst

```diff
@@ -57,6 +65,32 @@ on this instance. This means that:
 
 *   Replica IDs will change in accordance with the destination instance configuration.
 
+Authentication
+~~~~~~~~~~~~~~
+
+If authentication is required on the instance, use one of the following ways to pass the
```

Comment:
> If authentication is required on the instance

Looks like write access should always require authentication. I'd rephrase this and added a link to https://www.tarantool.io/en/doc/latest/book/admin/access_control/ (there are too many subsystems in Tarantool that require authentication :)).